### DISCUSSION TITLE
Thermostat manual temp override

### DISCUSSION THREAD

**Discussion by scantineau** [john30/ebusd#884] [Q&A] [answered]
Hello ! 

Do anybody knows in which topic should I publish to override the thermostat setpoint temp ?

My thermostat is a VRC 470.

Thanks :-)

**Comment by cyberthom42** [comment, marked as answer]
Have look here:
https://github.com/john30/ebusd-configuration/issues/234

**Reply by scantineau** [reply]
Hey ! Thank you for your answer !
